packages/docs/docs/concepts/user-components.md

```diff
@@ -212,89 +205,85 @@ const Toolbar = () => {
+## Defining editable elements
+Let's say we're building a Hero user component like so:
 
 ```jsx
-const Hero = ({title}) => {
+const Hero = ({background, title}) => {
   return (
-    <div>
-      <h2>{title}</h2>
-      <section>
-        <h3>Yo</h3>
-      </section>
-      I need some coffee to continue writing this
+    <div style={{ background }}>
+      <span>{title}</span>
       <section>
-        <h2>Hi</h2>
+        <h3>I need some coffee to continue writing this</h3>
       </section>
     </div>
   )
 }
 ```
 
-Previously, we discussed how `<Canvas />` creates a droppable Canvas node; that concept is applied universally in Craft.js. Hence, we just have to wrap our `section` elements in the example above with a `Canvas`. 
+Your first instinct is to simply just use the Text component directly like so:
 
-```jsx {5,9}
-const Hero = ({title}) => {
+```jsx
+const Hero = ({background, title}) => {
   return (
-    <div>
-      <h2>{title}</h2>
-      <Canvas id="Header" is="section">
-        <h3>Yo</h3>
-      </section>
-      I need some coffee to continue writing this
-      <Canvas id="Footer" is="section">
-        <h2>Hi</h2>
-      </section>
+    <div style={{ background }}>
+      <Text text={title} />
+      <section>... </section>
     </div>
   )
 }
 ```
 
-But wait, what if we want to define rules for these two new droppable regions? 
+But this won't really work - the Text Component will not have its own Node. Instead, it will still be a part of the Hero's Node. So for example, inside the Text Component, if we call `setProps(props => props.text = "...")`, it will actually be editing the props of `Hero` (in this case it will be adding a new prop `text` to Hero, which is not consumed by Hero anyway). Essentially, this means you can't have your users to click on the Text component and have them edit the text independently. 
 
-The `is` prop of the `<Canvas />` component is used to specify the `type` of the User Element for the Canvas Node. Hence, instead of specifying a simple DOM element which we don't have much control over, let's specify a brand new User Component.
 
+Remember how the `<Element / >` component was used to define/configure Nodes? Well, we can use that here to define a new Node for our Text component:
 
-```jsx {1-23,29,33}
-const HeroHeader = ({children}) => {
+
+```jsx
+const Hero = ({background, title}) => {
   return (
-    <section>
-      {children}
-    </section>
+    <div style={{ background }}>
+      <Element id="title" is={Text} text={title} />
+      <section>
+        <h3>I need some coffee to continue writing this</h3>
+      </section>
+    </div>
   )
 }
+```
 
-HeroHeader.craft = {
-  rules : {...}
-}
+> You must specify the `id` prop when defining new Nodes with `<Element />` inside a User Component
+
+We call these linked nodes since they are linked to another Node via an arbitary `id`. In this case, the `Text` node is linked to `Hero`'s node via its "title" id.
```

Comment:
"title" id sounds a bit confusing, almost like title prop serves as an id -> shall we change this id to be "title_text" just to make that slightly more obvious